packages/appstore/src/components/modals/wallets-upgrade-modal/wallets-upgrade-modal.scss

```diff
@@ -41,6 +60,12 @@
     border-radius: 0.8rem;
     background: var(--system-light-7-secondary-background, #f2f3f4);
```

Comment:
```suggestion
    background: var(--general-main-1);
```

Reply:
I found that if we don't provide a background color to the modal, it will default to the base color according to the light and dark.